_deployment/TdsConfigurationTasks

```diff
@@ -0,0 +1,8 @@
+---
+title: TDS Configuration Tasks
```

Comment:
I know it doesn't matter in the grand scheme of things but I preferred the `TDS Common Configuration Steps` rather than the title here.  All the things in this section are not required.